.travis.yml

```
@@ -1,2 +1,21 @@
 language: nix
-script: nix-build --attr proto3-suite-linux --arg enableDhall true release.nix
\ No newline at end of file
+
+script: nix-build --attr proto3-suite-linux --arg enableDhall true release.nix
+
+## Caching: https://github.com/travis-ci/travis-ci/issues/6604
+sudo: required
+
+install: |
+  if [ "$(ls -A $HOME/nix-cache)" ]; then
```

I know this code is taken from the above issue, but there are still a few things that I think we can fix:

```suggestion
  if [ -e ~/nix-cache ]; then
```